2.7/README.md

```diff
@@ -28,26 +28,19 @@ the nodejs itself is included just to make the npm work.
 
 Usage
 ---------------------
-To build a simple [python-sample-app](https://github.com/sclorg/s2i-python-container/tree/master/2.7/test/setup-test-app) application
-using standalone [S2I](https://github.com/openshift/source-to-image) and then run the
-resulting image with [Docker](http://docker.io) execute:
 
-*  **For RHEL7 based image**
-    ```
-    $ s2i build https://github.com/sclorg/s2i-python-container.git --context-dir=2.7/test/setup-test-app/ rhscl/python-27-rhel7 python-sample-app
-    $ docker run -p 8080:8080 python-sample-app
-    ```
+For this, we will assume that you are using the rhscl/python-27-rhel7 image, available via python:2.7 imagestream tag in Openshift.
```

Comment:
Could you distinguish technical names from the surrounding text please? I.e. onclose `rhscl/python-27-rhel7 image` and `python:2.7` in backticks for easier readability.

Reply:
Will do, thanks!